seed/seedwriter/writer.go

```diff
@@ -286,6 +302,13 @@ func New(model *asserts.Model, opts *Options) (*Writer, error) {
 	return w, nil
 }
 
+func initManifestFromOptions(opts *Options) *Manifest {
+	if opts.Manifest == nil {
+		return NewManifest()
+	}
+	return opts.Manifest
+}
```

Comment:
this could be a private method on Options

Reply:
Like this suggestion, done!